v3/src/components/case-table/use-white-space-click.ts

```diff
@@ -36,7 +36,7 @@ export function useWhiteSpaceClick({ gridRef }: IProps) {
     })
   }, [componentRef, data])
 
-  const handleWhiteSpaceClick = useCallback(() => {
+  const clearCurrentSelection = useCallback(() => {
```

Comment:
The reason I named this function `handleWhiteSpaceClick` originally is because a name like `clearCurrentSelection` implies rather strongly that calling it will clear the selection, but it doesn't actually change the selection under some very important circumstances. If you would like to have access to a function which actually clears the selection, then the `clearCaseSelection` function can be exported from this hook in addition to `handleWhiteSpaceClick`.